package.xml

```diff
@@ -9,16 +9,15 @@
 
   <buildtool_depend>cmake</buildtool_depend>
 
-  <depend>KimeraRPGO</depend>
-  <depend>GTSAM</depend>
-  <depend>gflags</depend>
-  <depend>glog</depend>
-  <depend>opencv</depend>
-  <depend>opengv</depend>
   <depend>dbow2</depend>
+  <depend>gtsam</depend>
+  <depend>kimera_rpgo</depend>
+  <depend>libgflags-dev</depend>
+  <depend>libgoogle-glog-dev</depend>
+  <depend>libopencv-dev</depend>
```

Comment:
Same, should be opencv  or rather `opencv3_catkin` if we use ETH's catkinized opencv3

Reply:
If we use libopencv-dev, what are we assuming? That ppl installed opencv how?

Reply:
This is the correct/official rosdep key name for the opencv3 library:

https://github.com/ros/rosdistro/blob/4bfca115e18c733686b3b39524fbf1e227177574/rosdep/base.yaml#L2815

By declaring the same key name as the dependency, users with other packages depending on opencv3 don't have to monkey patch there manifest to use the same key to avoid rosdep from installing libopencv-dev from apt repo if the kimera workspace it builds from already includes a source build of opencv.

Note how this is used in the Dockerfile that builds the underlay for the ros wrapper:
https://github.com/MIT-SPARK/Kimera-VIO-ROS/pull/32/files#diff-3254677a7917c6c01f55212f86c57fbfR71-R95